src/router.rs

```diff
@@ -25,9 +25,14 @@ impl<Data> Router<Data> {
         path: &'a str,
         method: &http::Method,
     ) -> Option<(&'a BoxedEndpoint<Data>, RouteMatch<'a>)> {
-        self.table
-            .route(path)
-            .and_then(|(r, p)| Some((r.endpoints.get(method)?, p)))
+        let route = self.table.route(path).unwrap();
```

Comment:
This `unwrap` will panic if the route isn't found. You should be able to replace it with `route(path)?`

Reply:
No worries!